app/templates/events/event_list.html

```diff
@@ -53,139 +62,134 @@ <h1 class="text-center">Events List for {{selectedTerm.description}}</h1>
-            <td nowrap>{{event.timeStart.strftime('%-I:%M %p')}} - {{event.timeEnd.strftime('%-I:%M %p')}}</td>
-            <td>{{event.location}}</td>
-            {% if user.isAdmin %}
-              <td>
-                {% if event.isPast %}
-                  {% set isPastEvent = "true" %}
-                {% endif %}
-                <button class="btn btn-primary" onclick="showEmailModal({{event.id}}, 'Unknown', {{selectedTerm}}, {{isPastEvent}})">
-                  <span class="bi bi-envelope-fill"> Invite</span>
-                </button>
-              </td>
-            {% else %}
-              <td>
-              {% if event.id in rsvpedEventsID %}
-                <button type="button" class="btn btn-danger" onclick="removeRsvpForEvent({{event.id}})">Remove RSVP</button>
-              {% elif (not event.isPast) and (event.isRsvpRequired) %}
+              <td nowrap>{{event.startDate.strftime('%m/%d/%Y')}}</td>
+              <td nowrap>{{event.timeStart.strftime('%-I:%M %p')}} - {{event.timeEnd.strftime('%-I:%M %p')}}</td>
+              <td>{{event.location}}</td>
+              {% if user.isAdmin %}
+                <td>
+                  {% if event.isPast %}
+                    {% set isPastEvent = "true" %}
+                  {% endif %}
+                  <button class="btn btn-primary" onclick="showEmailModal({{event.id}}, 'Unknown', {{selectedTerm}}, {{isPastEvent}})">
+                    <span class="bi bi-envelope-fill"> Invite</span>
+                  </button>
+                </td>
+              {% else %}
+                <td>
+                {% if event.id in rsvpedEventsID %}
+                  <button type="button" class="btn btn-danger" onclick="removeRsvpForEvent({{event.id}})">Remove RSVP</button>
+                {% elif (not event.isPast) and (event.isRsvpRequired) %}
                   {% set buttonName = "Join Waitlist" if (event.rsvpLimit) and (currentEventRsvpAmount[event.id] >= event.rsvpLimit) else "RSVP" %}
-                    <button type="button" class="btn btn-primary" onclick="rsvpForEvent({{event.id}})">{{buttonName}}</button>
+                      <button type="button" class="btn btn-primary" onclick="rsvpForEvent({{event.id}})">{{buttonName}}</button>
+                {% endif %}
+                </td>
               {% endif %}
-              </td>
+            </tr>
+          {% endif %}
+            {% if event.isPast %}
+              <tr class="table-light showlist" style="display: none;">
+                {% if type != "bonner" and type != "studentLed" %}
+                  {% if event.singleProgram%}
+                    <td  class="text-secondary">{{event.singleProgram.programName}}</td>
+                  {% else %}
+                    <td  class="text-secondary">This event does not belong to any program.</td>
+                  {% endif %}
+                {% endif %}
+                {% if user.isCeltsAdmin or user.isProgramManagerFor(event.singleProgram)%}
```

Comment:
Did this come from a merge conflict? `event.singleProgram` doesn't exist anymore, and should break in your code.